h2o-core/src/main/java/water/AutoBuffer.java

```diff
@@ -277,7 +277,8 @@ public AutoBuffer( InputStream is ) {
   private void checkVersion(String version) {
     final boolean ignoreVersion = Boolean.getBoolean(H2O_SYSTEM_SERIALIZATION_IGNORE_VERSION);
     if (! version.equals(H2O.ABV.projectVersion())) {
-      String msg = "Found version "+version+", but running version "+H2O.ABV.projectVersion();
+      String msg = "Found version "+version+", but running version "+H2O.ABV.projectVersion()
+              + ". If you wish to import older model into a newer version of H2O, please refer to the MOJO import functionality.";
```

Comment:
Can we turn this into an H2O technote, eg.https://0xdata.atlassian.net/browse/TN-1? (see H2O#technote)

Reply:
Added https://0xdata.atlassian.net/browse/TN-14